test/passing/tests/js_source.ml.ocp

```diff
@@ -3020,14 +3020,12 @@ type (_, _) eq = Refl : ('a, 'a) eq
 let magic : 'a 'b. 'a -> 'b =
   fun (type a b) (x : a) ->
   let module M =
-    (functor
-      (T : sig
-         type 'a t
-       end)
-      ->
-      struct
-        let f (Refl : (a T.t, b T.t) eq) = (x :> b)
-      end)
+    (functor (T : sig
+                type 'a t
+              end) ->
```

Comment:
It was better to have a break before the functor argument here.

Otherwise the whole formatting looks great!

Reply:
I've fixed this case.